### PR TITLE
Show the accuracy of perofrmance in quiz 

### PR DESCRIPTION
**Pull request by harishnavnit**
I think this would be a nice simple feature to have. What do you think ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,4 +21,5 @@
 *.app
 
 # Other files.
-*.user*
\ No newline at end of file
+*.user*
+build/*
```
Comment (martinrotter) [inline .gitignore]: Okay.

**Comment by martinrotter** [comment]
Displaying additional content in the Quiz template must match content which is trully visible in generated APK Android application. Try to generate any Quiz application, load it to phone and check if there is percentage also 

@croozeus What you think?

**Comment by croozeus** [comment]
We do show some stats at the end: Total correct, total wrong and unanswered.

Showing percentage can be added but not a priority right now. If it is added to Quiz, it should also be added to Flashcards and Learn Spellings template: both in the toolkit and the mobile application of the template.

**Comment by harishnavnit** [comment]
On Sun, Mar 8, 2015 at 1:46 PM, croozeus <notifications@github.com> wrote:

> Showing percentage can be added but not a priority right now.
>
Yeah, it's just a little feature that I thought would be cool to add. As a
kid, I'd be interested to see my score(or how many "stars", my percentage
score or Good/Bad etc). But yes, this is not a priority.

> If it is added to Quiz, it should also be added to Flashcards and Learn
> Spellings template: both in the toolkit and the mobile application of the
> template.
>
I can add the same for other templates too, but I really haven't checked
out the mobile application. I'll keep you posted. For now, you may close
this request.

Cheers,
R.Harish Navnit
The Enigma <http://harishnavnit.wordpress.com/>

**Pull request closed without merging**